definitions/infra-host/definition.yml

```diff
@@ -21,3 +21,17 @@ compositeMetrics:
 configuration:
   entityExpirationTime: DAILY
   alertable: true
+
+  #  SELECT entityGuid (target), nodeName (source) FRom Span where nodeName is not null
+  #  existing relationship
```

Comment:
The query is this `SELECT entityGuid as target, nodeName as source FRom Span where nodeName is not null` the target in this case is where the telemetry is coming from, but to me it makes most sense to declare the rule on the target because it can have a relationship between multiple types without having to be declared multiple times

As a consequence, navigation to find all rules with source|target as X will be more difficult.

Reply:
no harm either right? Also better mimics what nrql can do